test/controller/document_upload/remove.document.controller.spec.unit.ts

```diff
@@ -85,10 +87,27 @@ describe("remove document url tests", () => {
   it("should redirect to document upload page if No submitted", async () => {
     const res = await  request(app)
       .post(pageURLs.OBJECTIONS_REMOVE_DOCUMENT)
-      .set(REMOVE_DOCUMENT_FORM_FIELD, "no")
+      .send({[REMOVE_DOCUMENT_FORM_FIELD]: "no"})
+      .set("Referer", "/")
+      .set("Cookie", [`${COOKIE_NAME}=123`]);
+
+    expect(mockDeleteAttachment).toBeCalledTimes(0);
+    expect(res.status).toEqual(302);
+    expect(res.header.location).toEqual(OBJECTIONS_DOCUMENT_UPLOAD);
+  });
+
+  it("should redirect to document upload page and call objections service if Yes submitted", async () => {
+    const res = await  request(app)
+      .post(pageURLs.OBJECTIONS_REMOVE_DOCUMENT)
+      .send({
+        [REMOVE_DOCUMENT_FORM_FIELD]: "yes",
+        [ATTACHMENT_ID_FORM_FIELD]: ATTACHMENT_ID,
+        },
+      )
       .set("Referer", "/")
       .set("Cookie", [`${COOKIE_NAME}=123`]);
 
+    expect(mockDeleteAttachment).toBeCalledTimes(1);
```

Comment:
could you also check the correct attachment id is passed to the mockDeleteAttachment